flink-cyber/flink-indexing/flink-indexing-hive/src/main/java/com/cloudera/cyber/indexing/hive/tableapi/TableApiAbstractJob.java

```diff
@@ -172,7 +172,8 @@ protected TableDescriptor.Builder fillTableOptions(TableDescriptor.Builder build
                 .option("partition.time-extractor.timestamp-pattern", "$dt $hr:00:00")
                 .option("sink.partition-commit.trigger", "process-time")
                 .option("sink.partition-commit.delay", "1 h")
-                .option("sink.partition-commit.policy.kind", "metastore,success-file");
+                .option("sink.partition-commit.policy.kind", "metastore,success-file")
+                .option("hive.storage.file-format", "orc");
```

Comment:
Make sure to pull the latest updates from the CYB-175 branch

Reply:
Merged in latest from CYB-175.